sympy/solvers/diophantine.py

```diff
@@ -55,12 +55,12 @@ def diophantine(eq, param=symbols("t", integer=True)):
     >>> from sympy.solvers.diophantine import diophantine
     >>> from sympy.abc import x, y, z
     >>> diophantine(x**2 - y**2)
-    set([(-t_0, -t_0), (t_0, -t_0)])
+    set([(t_0, -t_0 + 1), (t_0, t_0 + 1)])
```

Comment:
how is that `+1` getting there if we are only ignoring sign? This is clearly wrong. Is it part of the `merge_terms` (or whatever it is named) process?